gradle-plugin/plugin/src/test/java/com/yelp/plugin/PluginTests.kt

```diff
@@ -32,4 +32,20 @@ class PluginTests {
                 .build()
         Assert.assertEquals(TaskOutcome.UP_TO_DATE, result2ndRun.task(":generateSwagger")?.outcome)
     }
+
+    @Test
+    fun testGradle5() {
```

Comment:
nit. I would rename this test to something like "testMinSupportdGradleVersion" or something similar, just to make sure that we'll not be surprised in the future about what is this test doing